Script/AtomicEditor/editor/Editor.ts

```diff
@@ -123,15 +143,12 @@ class Editor extends Atomic.ScriptObject {
 
     // event handling
     handleExitRequested(data) {
-        Preferences.getInstance().write();
-        EditorUI.shutdown();
-
-    }
-
-    handleEditorEventQuit(data) {
-
-        this.sendEvent("ExitRequested");
-
+      this.subscribeToEvent("AllResourceEditorsClosed", (_) => {
```

Comment:
I think what I was saying here is that the name should reflect that it is a shutdown event